docs/data/date-pickers/date-time-picker/CustomDateTimePicker.tsx

```diff
@@ -1,57 +1,32 @@
 import * as React from 'react';
-import dayjs, { Dayjs } from 'dayjs';
+import dayjs from 'dayjs';
 import AlarmIcon from '@mui/icons-material/Alarm';
 import SnoozeIcon from '@mui/icons-material/Snooze';
-import TextField from '@mui/material/TextField';
 import ClockIcon from '@mui/icons-material/AccessTime';
 import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
 import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
-import { DateTimePicker } from '@mui/x-date-pickers/DateTimePicker';
-import { MobileDateTimePicker } from '@mui/x-date-pickers/MobileDateTimePicker';
+import { Unstable_NextDateTimePicker as NextDateTimePicker } from '@mui/x-date-pickers/NextDateTimePicker';
 import Stack from '@mui/material/Stack';
 
 export default function CustomDateTimePicker() {
-  const [dateWithNoInitialValue, setDateWithNoInitialValue] =
-    React.useState<Dayjs | null>(null);
-  const [dateWithInitialValue, setDateWithInitialValue] =
-    React.useState<Dayjs | null>(dayjs('2019-01-01T18:54'));
-
   return (
     <LocalizationProvider dateAdapter={AdapterDayjs}>
       <Stack spacing={3}>
-        <DateTimePicker
+        <NextDateTimePicker
           openTo="hours"
-          value={dateWithInitialValue}
-          onChange={(newValue) => {
-            setDateWithInitialValue(newValue);
-          }}
+          defaultValue={dayjs('2022-04-07T15:30')}
           components={{
             LeftArrowIcon: AlarmIcon,
             RightArrowIcon: SnoozeIcon,
             OpenPickerIcon: ClockIcon,
           }}
-          renderInput={(params) => (
-            <TextField {...params} helperText="Hardcoded helper text" />
-          )}
-        />
-        <MobileDateTimePicker
-          value={dateWithInitialValue}
-          onChange={(newValue) => {
-            setDateWithInitialValue(newValue);
+          componentsProps={{
+            input: {
+              helperText: 'Hardcoded helper text',
+            },
           }}
-          label="With error handler"
-          onError={console.log}
-          inputFormat="YYYY/MM/DD hh:mm a"
-          mask="____/__/__ __:__ _M"
-          renderInput={(params) => <TextField {...params} />}
-        />
-        <DateTimePicker
-          value={dateWithNoInitialValue}
-          onChange={(newValue) => setDateWithNoInitialValue(newValue)}
-          renderInput={(params) => (
-            <TextField {...params} helperText="Clear Initial State" />
-          )}
         />
+        <NextDateTimePicker label="With error handler" onError={console.log} />
```

Comment:
Does it work for you? 🤔 
I get no console error whatever I enter in the field...

Reply:
If you add some validation it works
Without validation you can no longer easily type invalid date indeed

I'd be in favor of just dropping this example

Reply:
I'm in favour of this. 👍 
We can drop this example. 👌